buildSrc/subprojects/configuration/src/main/kotlin/org/gradle/gradlebuild/java/AvailableJavaInstallations.kt

```diff
@@ -119,8 +119,8 @@ open class AvailableJavaInstallations(project: Project, private val javaInstalla
         return mapOf(
             "Must set project or system property '$java7HomePropertyName' to the path of an $oracleJdk7, is currently unset." to (jdkForCompilation == null),
             validationMessage(java7HomePropertyName, jdkForCompilation, oracleJdk7) to (jdkForCompilation != null && jdkForCompilation.displayName != oracleJdk7),
-            "Must use Oracle JDK 8 to perform this build. Is currently ${currentJavaInstallation.displayName} at ${currentJavaInstallation.javaHome}." to
-                (currentJavaInstallation.displayName != oracleJdk8)
+            "Must use Oracle JDK 8/9 to perform this build. Is currently ${currentJavaInstallation.displayName} at ${currentJavaInstallation.javaHome}." to
```

Comment:
What about Java 10? Not much changed between 9 and 10 and 9 is no longer supported.

Reply:
Thanks for reviewing. We're going to migrate step by step instead of across two major release in one time.

Reply:
Thanks for the explanation.